### PR TITLE
docs: add doc for render blocks

### PR DESCRIPTION
**Pull request by Saul-Mirone**

### CODE REVIEW

File: packages/docs/render-blocks.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
+# Render Blocks
+
+In block suite, blocks can be rendered by any UI framework.
+A block should be rendered to a DOM element, and we use `view` to represent the renderer.
+
+By default, we provide a [lit](https://lit.dev/) renderer called `@blocksuite/lit`.
+But it's still possible to use other UI frameworks.
+I'll introduce later about how to write a custom renderer in [custom renderer](/#WIP).
```
Comment (doodlewind) [inline packages/docs/render-blocks.md:8]: Stick to using "we" should be the best practice.

Comment (Saul-Mirone) [reply packages/docs/render-blocks.md:8]: Done

File: packages/docs/render-blocks.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
+# Render Blocks
+
+In block suite, blocks can be rendered by any UI framework.
```
Comment (doodlewind) [inline packages/docs/render-blocks.md:3]: Please kindly stick to using the term "BlockSuite" in docs 😄

Comment (Saul-Mirone) [reply packages/docs/render-blocks.md]: Got it

**Review by doodlewind — APPROVED** [review]

**Pull request merged**